disable -Werror and hardcoded optimization flag
gcc -> cc
remove unsupported flags -fweb and -frename-registers
make -> gmake

Index: Quake/common.make
--- Quake/common.make.orig
+++ Quake/common.make
@@ -1,9 +1,9 @@
 HOST_OS =$(shell uname|sed -e s/_.*//|tr '[:upper:]' '[:lower:]')
 
-CC = gcc
+CC = cc
 # HOST_CC is for bintoc which is run on the host OS, not
 #         the target OS: cross-compile friendliness.
-HOST_CC = gcc
+HOST_CC = cc
 LINKER = $(CC)
 STRIP ?= strip
 WINDRES ?= windres
@@ -15,9 +15,9 @@ ifeq ($(USE_SDL3),1)
 CFLAGS += -DUSE_SDL3
 endif
 
-CHECK_GCC = $(shell if echo | $(CC) $(1) -Werror -S -o /dev/null -xc - > /dev/null 2>&1; then echo "$(1)"; else echo "$(2)"; fi;)
+CHECK_GCC = $(shell if echo | $(CC) $(1) -S -o /dev/null -xc - > /dev/null 2>&1; then echo "$(1)"; else echo "$(2)"; fi;)
 
-CFLAGS += -MMD -Wall -Wno-trigraphs -Werror -std=gnu11 -fno-common
+CFLAGS += -MMD -Wall -Wno-trigraphs -std=gnu11 -fno-common
 ifneq ($(HOST_OS),darwin)
 CFLAGS += -D_FILE_OFFSET_BITS=64
 endif
@@ -29,12 +29,9 @@ CFLAGS += -fno-omit-frame-pointer
 DO_STRIP=
 else
 DFLAGS += -DNDEBUG
-CFLAGS += -O3
 CFLAGS += -g
 CFLAGS += -fno-omit-frame-pointer
 #CFLAGS += $(call CHECK_GCC,-flto=auto,)
-CFLAGS += $(call CHECK_GCC,-fweb,)
-CFLAGS += $(call CHECK_GCC,-frename-registers,)
 #CFLAGS += $(call CHECK_GCC,-fno-asynchronous-unwind-tables,)
 CFLAGS += $(call CHECK_GCC,-fno-ident,)
 CMD_STRIP=$(STRIP) $(1)
@@ -283,7 +280,7 @@ OBJS := strlcat.o \
 	$(SYSOBJ_SYS) $(SYSOBJ_MAIN)
 
 $(BINTOC_EXE): ../Shaders/bintoc.c
-	make -B -C ../Misc/vq_pak
+	gmake -B -C ../Misc/vq_pak
 
 .SECONDARY:
 ../Shaders/Compiled/$(GLSLANG_OUT_FOLDER)/%_frag.spv: ../Shaders/%.frag
